testez-cli/src/tools/roblox_cli.rs

```diff
@@ -31,5 +37,10 @@ pub fn roblox_cli_run(place_path: &Path, entrypoint_path: &str, as_core_script:
         .status()
         .expect("failed to execute Roblox-CLI; is it installed?");
 
-    assert!(status.success(), "Roblox-CLI exited with an error.");
+    if !status.success() {
+        log::error!("Roblox-CLI exited with an error");
+        process::exit(1);
+    } else {
+        process::exit(0);
+    }
```

Comment:
I don't know if this fix is in quite the right spot. Does our panic handler exit with an error? That might be the place to fix this, since I don't think utility functions like this should be terminating the process.

Reply:
Oh, good point; I misunderstood the organization here. It looks like I'll need to introduce some panic-unwinding logic.